language/resource.language.fr_fr/strings.po

```diff
@@ -28,22 +28,22 @@ msgstr "Épisodes en cours"
 #: /1080i/Custom_1115_Dialog_Shortcuts.xml
 msgctxt "#31002"
 msgid "Edit $INFO[Container(22001).ListItem.Label,[LOWERCASE],[/LOWERCASE]] categories"
-msgstr ""
+msgstr "Editer la catgorie de $INFO[Container(22001).ListItem.Label,[LOWERCASE],[/LOWERCASE]]"
```

Comment:
Traductions telles quelles, vous devez les laisser ainsi, exemple 

Éditer $INFO[Container(22001).ListItem.Label,[LOWERCASE],[/LOWERCASE]] catégories

Reply:
Bonjour @icarok99,
J'aimerais bien voir un exemple en remplacent les variables pour valider le bon ordre des mots.  A première vue l'ordre proposé ne serait pas correct en français.

Reply:
This file of mine is updated with the latest updates.

https://github.com/icarok99/skin.arctic.fuse/blob/omega/language/resource.language.fr_fr/strings.po

Reply:
If I'm right by replacing the "listItem.label" variable with a more understandable word, initial english sentence will be :
Edit "item menu name" categories

So, the good french translation is :
Editer les catégories de "item menu name" 

There no doubt with this and ordering word differently won't be correct in french.
That's why in my proposition I put variable at the end of the sentence.

Is there a coding problem to put variable at the end ?

Reply:
It is not to translate this, Only what comes before $INFO and after [/LOWERCASE]]

$INFO[Container(22001).ListItem.Label,[LOWERCASE],[/LOWERCASE]]

Reply:
OK 
What I'm trying to explain to you is that the words order is not the same in french. 
You can't put "catégories" word at the end of the of the sentence.
It's simply not correct in french language.

Reply:
No arquivo strings vai tá assim

Éditer $INFO[Container(22001).ListItem.Label,[LOWERCASE],[/LOWERCASE]] catégories

But on Kodi it will show normally "Éditer catégories"

Reply:
@ludovik35 - I'm not opposed to the position of the `$INFO[]` moving if it is needed to make sense in the translation and it makes sense in the context of why the $INFO is used.

For these ones in the customisation screen, the $INFO is used to add the label of the item being edited in list 22001 to the option label. As an example: if  "Movies" item is highlighted, then the `$INFO[]` will output "movies" and label will display `Edit movies category` in English.

I take it in French the correct grammar would be the equivalent of `Edit the category of movies`. Every ListItem should have a label, so I think it is fine to rearrange these as you shouldn't ever end up with it reading `Edit the category of` with nothing at the end.

For other situations or properties it might be a different story, but in this case I think it should be safe to rearrange.

Reply:
Thanks @jurialmunkey, that's exactly answer to my question.
Don't worry, I always take time to understand $INFO usage in po files items before authorize myself to change order (if I think it's really necessary).